src/sentry/utils/performance_issues/detectors/large_payload_detector.py

```diff
@@ -54,6 +54,10 @@ def visit_span(self, span: Span) -> None:
             return
 
         payload_size_threshold = self.settings.get("payload_size_threshold")
+
+        if type(encoded_body_size) is str:
```

Comment:
i think isinstance is the more pythonic way of checking the type
```suggestion
        if isinstance(encoded_body_size, str):
```

Reply:
Thanks!